RELEASE_PROCESS.rst

```diff
@@ -7,9 +7,33 @@ Make the clone using:
 
 ::
 
-     git clone --branch master git@github.com:juju/juju-gui.git
-     cd juju-gui
-     git merge origin/develop
+    git clone git@github.com:juju/juju-gui.git gui-release
+    cd juju-gui
+
+Next you'll want to generate the list of changes for CHANGES.yaml.
+
+::
+
+    git log <last release>...HEAD --author 'jujugui' --format='* [%h] %b'
+
+Update CHANGES.yaml to include the list of changes--you can try to summarize
+the list of commits or just include them. If you do the latter, note that
+the list of commits may still need to be cleaned up to fit appropriately in the
+CHANGES file. Look at previous entries to see how it should look.
+
+Commit the changes to the changelog.
+
+::
+
+    git commit -am "Updating changelog."
```

Comment:
I'd split these out into two commands explicitly adding the separate files. I always have files in the tree that are not to be committed.